### PR TITLE
Made web_send_message_popup installable in 8.0

### PR DESCRIPTION
**Pull request by lbonjean**

### CODE REVIEW

**Comment by guewen** [comment]
tested, works fine, thanks
:+1:

File: web_send_message_popup/__openerp__.py [hunk 1]
```diff
@@ -44,6 +44,6 @@
  'website': 'http://www.camptocamp.com',
```
Comment (simahawk) [inline web_send_message_popup/__openerp__.py:44]: https://github.com/OCA/web

**Review by simahawk — CHANGES_REQUESTED** [review]
lint needs fixes

image

**Comment by simahawk** [comment]
@lbonjean would you mind to get back here for an easy fix?

**Pull request closed without merging**